src/concordances.cpp

```diff
@@ -0,0 +1,27 @@
+#include <Rcpp.h>
+using namespace Rcpp;
+
+int sign(const double& x){
+	return x == 0 ? 0 : x > 0 ? 1 : -1;
+}
+
+// Defined in Hollander, Wolfe & Chicken, Nonparametric Statistical Methods, 3ed., eq. (8.5), p. 394
+int Q(const double& a, const double& b, const double& c, const double& d) {
```

Comment:
Here you're overdoing the references :P See for example: https://stackoverflow.com/questions/3967177/when-to-use-const-and-const-reference-in-function-args

The `const` part is fine but the reference part (`&`) is not recommended when a type is "trivially copyable" (int, double, char). NumericVectors aren't trivially copyable because they may contain 1000s of numbers, but a single double is (because a pointer to it would basically be the same size).